src/lib/common/Format.h

```diff
@@ -31,6 +31,15 @@
 
 /* ****************************************************************************
 *
```

Comment:
The CHANGES_NEXT_RELEASE in the base branch (feature/722_unhardwire_format_cpr) already includes a mention to the format unhardwiring for CPrs forwarding. However, if we have included extra stuff regarding fixes in notifyFormat processing, an entry for that should also be included (in that case, we don't have an explicit issue number, but it is not mandatory to have it to include the entry in CHANGES_NEXT_RELEASE).